src/xdocs/config.xml

```diff
@@ -729,6 +729,33 @@
       </p>
     </section>
 
+    <section name="Id">
+      <p>
+        Each module has a <code>id</code> property that could be used to differentiate
+        two instances of the same Check. It is helpful to make different rules in suppressions
+        and make name of module instance to be defined by user.
```

Comment:
Replace last sentence with:
`This is custom name is required to be able to suppress violations of 1 of the checks in specific scenarios, while leaving the other unaffected. Without the custom module name, it is harder to differentiate one module's violations from the other.`

Reply:
You are right, should be no `is`.